Source/Scene/AutoExposure.js

```diff
@@ -0,0 +1,416 @@
+define([
+        '../Core/Color',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/PixelFormat',
+        '../Renderer/ClearCommand',
+        '../Renderer/Framebuffer',
+        '../Renderer/PixelDatatype',
+        '../Renderer/Sampler',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap'
+    ], function(
+        Color,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        PixelFormat,
+        ClearCommand,
+        Framebuffer,
+        PixelDatatype,
+        Sampler,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap) {
+    'use strict';
+
+    /**
+     * A post process stage that will get the luminance value at each pixel and
+     * uses parallel reduction to get the average luminance in a 1x1 texture.
+     * This texture can be used as input for tone mapping.
+     *
+     * @constructor
+     * @private
+     */
+    function AutoExposure() {
```

Comment:
I know this is private, but should it be `AutomaticExposure`?  Or is "AutoExposure" such a common abbreviation that we can allow it like WGS84?